Kinvey.Core/Store/DataStore.cs

```diff
@@ -345,14 +345,41 @@ public async Task<T> SaveAsync(T entity, CancellationToken ct = default(Cancella
 			return await request.ExecuteAsync();
 		}
 
-
-		/// <summary>
-		/// Deletes the entity associated with the provided id
+        /// <summary>
+		/// Saves specified entities to a Kinvey collection.
 		/// </summary>
-		/// <returns>The async task.</returns>
-		/// <param name="entityID">The Kinvey ID of the entity to delete.</param>
-		/// <param name="ct">[optional] CancellationToken used to cancel the request.</param>
-		public async Task<KinveyDeleteResponse> RemoveAsync(string entityID, CancellationToken ct = default(CancellationToken))
+		/// <returns>An async task with the <see cref="Kinvey.KinveyDataStoreResponse{T}"/> result.</returns>
+		/// <param name="entities">Entities to save.</param>
+		/// <param name="ct">[optional] CancellationToken used to cancel a request.</param>
+        public async Task<KinveyDataStoreResponse<T>> SaveAsync(IList<T> entities, CancellationToken ct = default(CancellationToken))
+        {
+            if (!int.TryParse(KinveyClient.ApiVersion, out int apiVersion) || apiVersion < 5)
+            {
+                throw new KinveyException(EnumErrorCategory.ERROR_GENERAL, EnumErrorCode.ERROR_DATASTORE_NOT_COMPATIBLE_KINVEY_API_VERSION, string.Empty);
+            }
+
+            if (entities == null || entities.Count == 0)
+            {
+                throw new KinveyException(EnumErrorCategory.ERROR_GENERAL, EnumErrorCode.ERROR_DATASTORE_EMPTY_ARRAY_OF_ENTITIES, string.Empty);
+            }
+
+            if(entities.Count > Constants.NUMBER_LIMIT_OF_ENTITIES)
```

Comment:
This is ok for now, but in the future we need to have a way to handle more than the entity limit, by making multiple multi-insert requests and collecting all the results before returning.